llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -7217,8 +7217,7 @@ void SelectionDAG::canonicalizeCommutativeBinop(unsigned Opcode, SDValue &N1,
 SDValue SelectionDAG::getNode(unsigned Opcode, const SDLoc &DL, EVT VT,
                               SDValue N1, SDValue N2, const SDNodeFlags Flags) {
   assert(N1.getOpcode() != ISD::DELETED_NODE &&
-         N2.getOpcode() != ISD::DELETED_NODE &&
-         "Operand is DELETED_NODE!");
+         N2.getOpcode() != ISD::DELETED_NODE && "Operand is DELETED_NODE!");
```

Comment:
Only leftover unrelated formatting changes in this file

Reply:
Okay, reverted the clang-format changes.